app/controllers/home_controller.rb

```diff
@@ -1,5 +1,6 @@
 class HomeController < ApplicationController
   def show
+    #passing this variable to the views?
     @client_id = ENV["client_id"]
```

Comment:
@maxglassie I think that makes sense. I would make sure *not* to put your `client_secret` into the view. You might also look into OmniAuth. Decent tutorial [here](https://www.sitepoint.com/rails-authentication-oauth-2-0-omniauth/).